pkgs/development/tools/glpaper/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchhg
+, meson, ninja
+, pkg-config
+, wlroots, wayland, wayland-protocols
+, pixman, libxkbcommon
+,libX11
+, libGL, libglvnd
+}:
+
+stdenv.mkDerivation {
+  name = "glpaper";
+  version = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
+
+  src = fetchhg {
+    url = "https://hg.sr.ht/~scoopta/glpaper";
+    rev = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
+    sha256 = "04y12910wvhy4aqx2sa63dy9l6nbs7b77yqpdhc96x2b3mgzgjfs";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config ];
+  buildInputs = [
+    wlroots wayland wayland-protocols wlroots
```

Comment:
There are few instances of weird spacing/line wrapping. Perhaps use nixpkgs-fmt.

Reply:
Also wlroots is here twice.

Reply:
Fixed, thanks!